parser_library/src/context/ordinary_assembly/ordinary_assembly_context.cpp

```diff
@@ -54,7 +54,11 @@ bool ordinary_assembly_context::create_symbol(
     return ok;
 }
 
-void ordinary_assembly_context::add_symbol_reference(symbol sym) { symbol_refs_.try_emplace(sym.name, std::move(sym)); }
+void ordinary_assembly_context::add_symbol_reference(symbol sym)
+{
+    auto name = sym.name;
+    symbol_refs_.try_emplace(name, std::move(sym));
```

Comment:
I still think that the copy is not needed. But this code will make two copies...